analyze_nplus_R33_directives.gen.go

```diff
@@ -1660,8 +1660,8 @@ var nginxPlusR33Directives = map[string][]uint{
         ngxMgmtMainConf | ngxConfTake1,
     },
     "ssl_ocsp": {
-        ngxHTTPMainConf | ngxHTTPSrvConf | ngxConfFlag,
-        ngxStreamMainConf | ngxStreamSrvConf | ngxConfFlag,
+        ngxHTTPMainConf | ngxHTTPSrvConf | ngxConfTake1,
```

Comment:
did we update these `.gen.go` via the code generator or by hand?

Reply:
the generator has a config file where we can override what gets detected, e.g. https://github.com/nginxinc/nginx-go-crossplane/blob/main/scripts/generate/configs/nplus_R33_config.json#L41

Recommend making the change there too, so the next time the generator gets run we won't lose this fix.